codechecker_lib/analyzers/config_handler_clang_tidy.py

```diff
@@ -91,16 +80,18 @@ def get_checker_configs(self):
             return res
 
         try:
+            # Match for clang tidy analyzer names and attributes.
+            clang_tidy_checker_rex = r'^(?P<checker_name>([^.]+))' \
+                                     r'\.(?P<checker_attribute>([^.]+))$'
+
+            tidy_pattern = re.compile(clang_tidy_checker_rex)
             tidy_config = json.loads(args.tidy_config)
             for checker_option in tidy_config.get('CheckOptions', []):
                 value = checker_option['value']
-                key_values = re.match(clangsa_pattern, checker_option['key'])
+                # We only store configs related to tidy checks. We run static
```

Comment:
Because we run the static analyzer separately, and the SA related configs in the tidy config file will not have any effect.